content/en/resources/keps/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+linktitle: Enhancements
+title: Kubernetes Enhancement Proposals (KEPs)
+description: |
+  List of enhancements that are proposed
```

Comment:
```suggestion
   List of Kubernetes enhancements.
```
“proposed” doesn't capture that some of these are fully implemented.

Reply:
In essence, all enhancements are proposals in various states of implementation, but I agree that the language can create some confusion.